lang/syn/src/codegen/accounts/constraints.rs

```diff
@@ -686,7 +689,7 @@ fn generate_constraint_init_group(
 
             let create_account = generate_create_account(
                 field,
-                quote! {anchor_spl::token::Mint::LEN},
+                quote! {anchor_spl::token_2022::Mint::LEN},
```

Comment:
`anchor_spl::token_2022::Mint::LEN` does not exist

Reply:
Bah, sorry, this went through a few iterations while I was cleaning it up, and I forgot to add the token_2022 types back in. I'll rewrite the spl token-2022 commit to add it